CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [unreleased]
+### Security
+### Added
+### Changed
+### Removed
+
+## [v0.7.0] - 26/11/2021
+### Security
+- dependencies update ([#6](https://github.com/flatcar-linux/locksmith/pull/6))
+- etcd, dbus and systemd Go dependencies update ([#8](https://github.com/flatcar-linux/locksmith/pull/8))
+
+### Added
+
+### Changed
+- ignore nil sent by dbus.Signal in the daemon ([#1](https://github.com/flatcar-linux/locksmith/pull/1))
+- import path to `flatcar-linux` ([#2](https://github.com/flatcar-linux/locksmith/pull/2))
+- Glide to Go modules ([#4](https://github.com/flatcar-linux/locksmith/pull/4))
+- locksmithd will not reboot outside of reboot windows ([#10](https://github.com/flatcar-linux/locksmith/pull/10))
+- endpoints reliability ([#11](https://github.com/flatcar-linux/locksmith/pull/11))
```

Comment:
This is not user facing, as https://github.com/flatcar-linux/locksmith/commit/1b5eb50 has not been tagged yet, I think we can remove this.